src/vehicle.cpp

```diff
@@ -1237,6 +1243,8 @@ void vehicle::use_controls()
         break;
     case control_cancel:
         break;
+    case toggle_scoop:
+        scoop_on=!scoop_on;
```

Comment:
You really don't like spaces, do you? This looks a operator `=!` - what's that? Oh, wait it's actually two operators: `scoop_on = !scoop_on;` - my fault.